### ISSUE TITLE
FutureWarning

### ISSUE THREAD

**Issue by siebrenf** [vanheeringen-lab/gimmemotifs#298]
https://github.com/vanheeringen-lab/gimmemotifs/blob/ecd4806eff4567744016f96b596d72e25a006e4a/gimmemotifs/maelstrom/__init__.py#L235

throws
```
FutureWarning: Attribute `affinity` was deprecated in version 1.2 and will be removed in 1.4. Use `metric` instead
```

[Docs](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.FeatureAgglomeration.html) suggest it might be more than just changing the keyword. Might have to become `metric=scipy.metrics.distance.correlation`
